plutus-use-cases/src/Language/Plutus/Coordination/Contracts/CrowdFunding.hs

```diff
@@ -11,6 +11,7 @@
 {-# LANGUAGE TemplateHaskell     #-}
 {-# LANGUAGE ViewPatterns        #-}
 {-# OPTIONS -fplugin=Language.Plutus.CoreToPLC.Plugin -fplugin-opt Language.Plutus.CoreToPLC.Plugin:dont-typecheck #-}
+{-# OPTIONS_GHC -Wno-overlapping-patterns #-}
```

Comment:
Is this due to use of custom patterns?

Reply:
It's due to a redundant pattern match in l. 140 - `_ -> False` - if you delete that you don't need to disable the warning. Also the `ViewPatterns` extension isn't necessary (anymore)

Reply:
Thanks, I'll do that. I didn't want to get into hunting down warnings in this PR, especially since it unhelpfully appears at the start of the TH block.

Reply:
Fixed.